File: Submissions/T15/H2.py
baris, kolom = map(int, input("Baris & Kolom : ").split())

matriks = []

for i in range(baris):
    data = list(map(int, input(f"Baris {i+1} : ").split()))
    matriks.append(data)

print(f"\nMatriks ({baris}x{kolom}) :")

for row in matriks:
    print(*row)

print(f"Transpose ({kolom}x{baris}) :")

for j in range(kolom):
    for i in range(baris):
        print(matriks[i][j], end=" ")
    print()

File: Submissions/T15/H2.cpp
#include <stdio.h>

int main() {
    int baris, kolom;

    printf("Baris & Kolom : ");
    scanf("%d %d", &baris, &kolom);

    int m[baris][kolom];

    for(int i=0;i<baris;i++){
        printf("Baris %d : ", i+1);

        for(int j=0;j<kolom;j++){
            scanf("%d",&m[i][j]);
        }
    }

    printf("\nMatriks (%dx%d) :\n", baris, kolom);

    for(int i=0;i<baris;i++){
        for(int j=0;j<kolom;j++){
            printf("%d ", m[i][j]);
        }
        printf("\n");
    }

    printf("Transpose (%dx%d) :\n", kolom, baris);

    for(int j=0;j<kolom;j++){
        for(int i=0;i<baris;i++){
            printf("%d ", m[i][j]);
        }
        printf("\n");
    }

    return 0;
}
